exercises/sublist/sublist_test.py

```diff
@@ -1,134 +1,137 @@
-            check_lists([1, 1, 2], [0, 1, 1, 1, 2, 1, 2]),
-            SUBLIST
+            sublist([1, 1, 2], [0, 1, 1, 1, 2, 1, 2]),
+            "sublist"
         )
 
     def test_sublist_at_start(self):
         self.assertEqual(
-            check_lists([0, 1, 2], [0, 1, 2, 3, 4, 5]),
-            SUBLIST
+            sublist([0, 1, 2], [0, 1, 2, 3, 4, 5]),
+            "sublist"
         )
 
     def test_sublist_in_middle(self):
         self.assertEqual(
-            check_lists([2, 3, 4], [0, 1, 2, 3, 4, 5]),
-            SUBLIST
+            sublist([2, 3, 4], [0, 1, 2, 3, 4, 5]),
+            "sublist"
         )
 
     def test_sublist_at_end(self):
         self.assertEqual(
-            check_lists([3, 4, 5], [0, 1, 2, 3, 4, 5]),
-            SUBLIST
+            sublist([3, 4, 5], [0, 1, 2, 3, 4, 5]),
+            "sublist"
         )
 
     def test_at_start_of_superlist(self):
         self.assertEqual(
-            check_lists([0, 1, 2, 3, 4, 5], [0, 1, 2]),
-            SUPERLIST
+            sublist([0, 1, 2, 3, 4, 5], [0, 1, 2]),
+            "superlist"
         )
 
     def test_in_middle_of_superlist(self):
         self.assertEqual(
-            check_lists([0, 1, 2, 3, 4, 5], [2, 3]),
-            SUPERLIST
+            sublist([0, 1, 2, 3, 4, 5], [2, 3]),
+            "superlist"
         )
 
     def test_at_end_of_superlist(self):
         self.assertEqual(
-            check_lists([0, 1, 2, 3, 4, 5], [3, 4, 5]),
-            SUPERLIST
+            sublist([0, 1, 2, 3, 4, 5], [3, 4, 5]),
+            "superlist"
         )
 
     def test_first_list_missing_element_from_second_list(self):
         self.assertEqual(
-            check_lists([1, 3], [1, 2, 3]),
-            UNEQUAL
+            sublist([1, 3], [1, 2, 3]),
+            "unequal"
         )
 
     def test_second_list_missing_element_from_first_list(self):
         self.assertEqual(
-            check_lists([1, 2, 3], [1, 3]),
-            UNEQUAL
+            sublist([1, 2, 3], [1, 3]),
+            "unequal"
         )
 
     def test_order_matters_to_a_list(self):
         self.assertEqual(
-            check_lists([1, 2, 3], [3, 2, 1]),
-            UNEQUAL
+            sublist([1, 2, 3], [3, 2, 1]),
+            "unequal"
         )
 
     def test_same_digits_but_different_numbers(self):
         self.assertEqual(
-            check_lists([1, 0, 1], [10, 1]),
-            UNEQUAL
+            sublist([1, 0, 1], [10, 1]),
+            "unequal"
         )
 
     # additional track specific test
     def test_inner_spaces(self):
         self.assertEqual(
-            check_lists(['a c'], ['a', 'c']),
-            UNEQUAL
+            sublist(['a c'], ['a', 'c']),
+            "unequal"
         )
 
     # additional track specific test
     def test_large_lists(self):
         l1 = list(range(1000)) * 1000 + list(range(1000, 1100))
         l2 = list(range(900, 1050))
-        self.assertEqual(check_lists(l1, l2), SUPERLIST)
```

Comment:
This was inconsistent with the rest as far as indentation goes, so I corrected it.